sdks/python/apache_beam/runners/portability/fn_api_runner/fn_runner.py

```diff
@@ -179,6 +178,13 @@ def run_pipeline(self,
       self._default_environment = environments.SubprocessSDKEnvironment(
           command_string=command_string)
 
+    if running_mode == 'in_memory' and self._num_workers != 1:
+      _LOGGER.warning(
+          'If direct_num_workers is not equal to 1, direct_running_mode '
```

Comment:
Maybe also log running_mode and self._num_workers as part of the message.

Reply:
Done.